bower.json

```diff
@@ -17,8 +17,8 @@
     "ez-js-rest-client": "ezsystems/ez-js-rest-client",
     "yui3": "http://yui.zenfs.com/releases/yui3/yui_3.18.1.zip",
     "flag-icon-css": "~0.6.4",
-    "alloy-editor": "ezsystems/alloy-editor#~0.6.0",
-    "widget": "http://download.ckeditor.com/widget/releases/widget_4.5.3.zip",
-    "lineutils": "http://download.ckeditor.com/lineutils/releases/lineutils_4.5.3.zip"
+    "alloy-editor": "liferay/alloy-editor#~0.7.5",
+    "widget": "http://download.ckeditor.com/widget/releases/widget_4.5.5.zip",
```

Comment:
Seems to be no changes actually: https://dev.ckeditor.com/query?status=closed&component=UI+%3A+Widgets&milestone=CKEditor+4.5.6&group=status&col=id&col=summary&col=component&col=owner&col=type&col=priority&col=version&order=priority